File: PlaceYourBets/FixtureList.vb
Public Class FixtureList

    Public Property stock() As List(Of Fixture)
        Get
            Return m_stock
        End Get
        Set(value As List(Of Fixture))
            m_stock = value
        End Set
    End Property
    Private m_stock As List(Of Fixture)
    Public Property success() As Integer
        Get
            Return m_success
        End Get
        Set(value As Integer)
            m_success = value
        End Set
    End Property
    Private m_success As Integer

End Class

File: PlaceYourBets.ConvertedToC#/FixtureList.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;
using System.Linq;
using System.Xml.Linq;
namespace PlaceYourBets
{
	public class FixtureList
	{

		public List<Fixture> stock {
			get { return m_stock; }
			set { m_stock = value; }
		}
		private List<Fixture> m_stock;
		public int success {
			get { return m_success; }
			set { m_success = value; }
		}

		private int m_success;
	}
}
